ChopChop/Views/Recipes/RecipeView.swift

```diff
@@ -2,6 +2,8 @@ import SwiftUI
 
 struct RecipeView: View {
     @ObservedObject var viewModel: RecipeViewModel
+    @State private var showSessionRecipe = false
+    @State private var showRecipeForm = false
```

Comment:
Maybe states can be stored in the view model instead as a published property, for separation of concerns.